keras_cv/layers/preprocessing/auto_contrast.py

```diff
@@ -31,7 +31,8 @@ class AutoContrast(VectorizedBaseImageAugmentationLayer):
 
     Args:
         value_range: the range of values the incoming images will have.
-            Represented as a two number tuple written [low, high].
+            Represented as a two number tuple written [low, high]. 
+            It can be passed as any datatype that returns an integer on indexing like a list, tuple, numpy array or tf.tensor
```

Comment:
I'm not sure this change is actually helpful - I think it just adds noise to end users.

Reply:
A more valuable update would be an example:

"""
i.e. `value_range=(0, 255)`
"""

Reply:
I thought in case users pass the input as a different datatype than a tuple, they must be aware that it actually works. 
About the example update, I will try to add it as soon as this conversation is resolved.

Reply:
@LukeWood Kindly reply if you still feel there is an issue.